packages/codemirror-graphql/src/utils/info-addon.ts

```diff
@@ -33,7 +33,7 @@ CodeMirror.defineOption(
   },
 );
 
-function createState(options: GraphQLInfoOptions) {
+function createState(options: GraphQLInfoOptions | true | (() => any)) {
```

Comment:
<img width="759" alt="image" src="https://github.com/graphql/graphiql/assets/7361780/24c5aca1-34c4-48c4-a990-edd151e81b69">

Reply:
also added the missing type for function while checking `options instanceof Function`